### PR TITLE
chore: Prepare release 2.20.1

### PR DESCRIPTION
**Pull request by LeonLuttenberger**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubCodeBuild8756EF16-4rfo0GHQ0u9a
* Commit ID: 9615ec80c921b2258b4ad6070fa1bc1ca65dcf5e
* Result: FAILED
* [Build Logs](https://gqgcj9s2gc.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=e4f0ef64-7449-41ea-b8bf-887133918062%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: docs/source/install.rst [hunk 1]
```diff
@@ -100,7 +100,7 @@ Here is an example of how to create and use the AWS SDK for pandas Lambda layer
           "awssdkpandas-layer",
           location=sam.CfnApplication.ApplicationLocationProperty(
             application_id="arn:aws:serverlessrepo:us-east-1:336392948345:applications/aws-sdk-pandas-layer-py3-8",
-            semantic_version="2.20.0",  # Get the latest version from https://github.com/aws/aws-sdk-pandas/releases
+            semantic_version="2.20.1",  # Get the latest version from https://github.com/aws/aws-sdk-pandas/releases
```
Comment (jaidisido) [inline docs/source/install.rst:103]: I don't think we will be creating a new SAR for a minor version so would bring it back to 2.20 perhaps?

File: docs/source/layers.rst [hunk 2]
```diff
@@ -2,7 +2,7 @@
 AWS Lambda Managed Layers
 ==========================
 
-Version 2.20.0
+Version 2.20.1
```
Comment (jaidisido) [inline docs/source/layers.rst:5]: Same for layers

**Review by malachi-constant — APPROVED** [review]

File: docs/source/install.rst (hunk 1 above)
Comment (LeonLuttenberger) [reply docs/source/install.rst:103]: As discussed on Slack, we'll still publish layers for this patch. Although it's a very small patch, the issue its addressing can have a potentially large customer impact.

**Pull request merged**